pyproject.toml

```diff
@@ -2,6 +2,10 @@
 name = "func-python"
 version = "0.2.0"
 description = "Knative Functions Python Middleware"
+packages = [
+    { include = "func_http", from = "src" },
+]
+
```

Comment:
The reason the folder is named `func_python` is such that it matches the package name as published on PyPI `func-python`.  Hence the reason there's no need for an explicit `packages` section in the pyproject.toml